.mailmap

```diff
@@ -20,45 +39,92 @@ Daniel Pollard <daniel@learnweaver.com>
+Eva Herrada <eva.herrada@adafruit.com> <dylan.herrada@adafruit.com>
+Eva Herrada <eva.herrada@adafruit.com> <dylan.herrada@gmail.com>
+Eva Herrada <eva.herrada@adafruit.com> dherrada <=>
+Florin Maticu <florin.maticu@gmail.com>
+Florin Maticu <florin.maticu@gmail.com> <3575408+flom84@users.noreply.github.com>
+Florin Maticu <florin.maticu@gmail.com> <flomaker84@gmail.com>
+Frédéric Pierson <fpierson@garatronic.fr>
+Fábio Souza <fs.embarcados@gmail.com>
 George Waters <gwatersdev@gmail.com>
 George Waters <gwatersdev@gmail.com> <george@georgeh2os.com>
+Glenn Moloney <glenn.moloney@gmail.com>
 Ha Thach <thach@tinyusb.org>
 Henrik Sölver <henrik.solver@gmail.com>
+Ihor Nehrutsa <Ihor.Nehrutsa@gmail.com>
+Ihor Nehrutsa <Ihor.Nehrutsa@gmail.com> <IhorNehrutsa@gmail.com>
 Ilya Dmitrichenko <errordeveloper@gmail.com>
 Ilya Dmitrichenko <errordeveloper@gmail.com> <ilya@xively.com>
+James Bowman <jamesb@excamera.com>
+James Bowman <jamesb@excamera.com> <James Bowman>
+James Carr <lesamouraipourpre@gmail.com>
+James Carr <lesamouraipourpre@gmail.com> <70200140+lesamouraipourpre@users.noreply.github.com>
+James Nadeau <james.nadeau@gmail.com>
+Jan Hrudka <jahr@atlas.cz>
 Jason Pecor <14111408+jpecor@users.noreply.github.com>
 Jeff Epler <jepler@gmail.com>
 Jeff Epler <jepler@gmail.com> <jeff@adafruit.com>
 Jeff Epler <jepler@gmail.com> <jepler@de11.u>
 Jeff Epler <jepler@gmail.com> <jepler@unpythonic.net>
+Jensen Kuras <jensechu@gmail.com>
+Jeremy Littler <brainboardz@gmail.com>
+Jeremy Littler <brainboardz@gmail.com> <87398149+BrainBoardz@users.noreply.github.com>
 Jerry Needell <jerryneedell@gmail.com>
 Joe Bakalor <jmbakalor@gmail.com>
+Jonah Yolles-Murphy <TGTechie01@gmail.com>
+Jonah Yolles-Murphy <TGTechie01@gmail.com> <39284876+TG-Techie@users.noreply.github.com>
+Jonah Yolles-Murphy <TGTechie01@gmail.com> <TGTechie01@gmial.com>
+Jonah Yolles-Murphy <TGTechie01@gmail.com> <tgtechie01@gmail.com>
+Jonathan Giles <jonathangiles@fastmail.fm>
+Jonathan Giles <jonathangiles@fastmail.fm> <jgiles@Jonathans-Air.lan>
+Jonny Bergdahl <jonny@bergdahl.it>
+Jonny Bergdahl <jonny@bergdahl.it> <bergdahl@users.noreply.github.com>
+Jos Verlinde <jos_verlinde@hotmail.com>
+Jos Verlinde <jos_verlinde@hotmail.com> <jos.verlinde@microsoft.com>
+Jos Verlinde <jos_verlinde@hotmail.com> <josverl@microsoft.com>
 Josh Klar <josh@klar.sh>
 Josh Klar <josh@klar.sh> <j@iv597.com>
 Juan Biondi <juanernestobiondi@gmail.com>
 Juan Biondi <juanernestobiondi@gmail.com> <juanernestobiondi@hotmail.com>
+Julia Hathaway <julia.hathaway@nxp.com>
 KalbeAbbas <kalbeabbas142@gmail.com>
 KalbeAbbas <kalbeabbas142@gmail.com> <kalbeabbas@142@gmail.com>
 Kamil Tomaszewski <kamil.tomaszewski@sony.com>
 Kamil Tomaszewski <kamil.tomaszewski@sony.com> <46525824+kamtom480@users.noreply.github.com>
-Kattni <kattni@adafruit.com> <kattni@kittyfish.org>
 Kattni Rembor <kattni@adafruit.com>
+Kattni Rembor <kattni@adafruit.com> <kattni@kattni.com>
```

Comment:
Kattni is no longer working with Adafruit so I assume her @kattni.com email would be better.

Reply:
Ah, right, good catch. Fixed.